### ISSUE TITLE
Accessibility work for academic calendar list

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#7858]
```[tasklist]
### Accessibility tasks
- [x] Setup callback to announce calendar date changes in the block like https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/uiowa_hours/src/Form/HoursFilterForm.php#L132
- [x] ensure wcag 2.1 aa accessibility - test on voiceover - webaim/siteimprove/lighthouse/ainspector page scans - voiceover guide https://dequeuniversity.com/assets/pdf/screenreaders/voiceover-macos-guide.pdf
```

**Comment by briand44** [comment]
Grooming conversation... Should this be blocked by #7854. Some could maybe be done prior to but should probably wait for that work to be done so we can tackle this all at once. If we run into any complex accessibility issues while working on this we can discuss and potentially create a separate slice if needed.

**Comment by joewhitsitt** [comment]
I forgot this was a separate slice and sprinkled in some Drupal.announce items into the JS as part of https://github.com/uiowa/uiowa/issues/7854

**Comment by bspeare** [comment]
The first issue that I am noticing is that Safari sets the date input default to today so that it doesn't show any results: https://registrar.prod.drupal.uiowa.edu/calendars-and-deadlines/academic-calendar.   Perhaps something that we missed previously testing date inputs for https://safety.uiowa.edu/crime-log?  Between that and https://adrianroselli.com/2019/07/maybe-you-dont-need-a-date-picker.html maybe we might want to consider removing the date filters?  The granularity of the filter is nice to have but the Session filter is probably the primary mode for filtering.

**Comment by bspeare** [comment]
Testing observations checklist:
- [x] Would a search and reset button be useful here?
  - [x] Forms must use input[type="submit"] or other form control buttons for submitting forms. 
- [ ] Using the Search field will filter the list but the voiceover will only update if you backspace and type.  Do we want to sprinkle in additional details for searchTerm only?  Something like `Drupal.announce('Filtering events for search term: ' + this.searchTerm)`?
- [x] Card titles need to be `<h3>`
- [x] Date input https://github.com/uiowa/uiowa/issues/7858#issuecomment-2284268900
  - [x] Safari sets the date input default to today 
- [x] Can't navigate Chosen categories with keyboard and voiceover - does this need to be a multi-select option?
   - [x] WAVE/Lighthouse error for Chosen - Missing form label - A form control does not have a corresponding label. "input.chosen-search-input" Add label to textbox control
   - [x] Violation - list widget does not contain one or more of following required owned elements with a role of: listitem. chosen-results 
   - [x] There are some patches we could add to make this more accessible.  See `https://github.com/department-of-veterans-affairs/LEAF/pull/714/files` as an example. 
- [ ] AI Inspector - Violation - Move div element into an appropriate landmark. (This may require creating an additional landmark.) drupal-live-announce - https://www.drupal.org/project/drupal/issues/3098857

**Comment by bspeare** [comment]
I was able to get the date input fields working in Safari, however, after reading through https://adrianroselli.com/2019/07/maybe-you-dont-need-a-date-picker.html it does seems like the date input field should be avoided if possible.   Filtering by month does help simplify the user interaction quite a bit.

**Comment by pyrello** [comment]
Adding this here from #7984:
Josh said we can remove the end date filter. I think Ben hid this in the accessibility work. We should probably remove this from the code if it's going to stay this way.
